cds-snc/cppd-medical-report
# Section 4: Updated content keys and FR translation

# Description

[Trello card:](https://trello.com/c/a46aCfHK/309-people-can-access-prototype-in-en-and-fr) Translates section 4 of the medical report into French

# Test Instructions

1. Go to the application
1. If your browser is in EN-xx locale, click `Français` at the top
1. Navigate through the intro section, Section 1, Section 2, and Section 3 to get to Section 4 (alternately, change the URL to `/fr/frsection4`)
1. See French translation (original EN and FR forms are linked in the [Trello card](https://trello.com/c/a46aCfHK/309-people-can-access-prototype-in-en-and-fr))

# Help Requested

If there are any things you spot that are issues we should come back to, but are out of scope of just the direct translation, please add a new Trello card or [use this one to quickly add a note to the checklist](https://trello.com/c/WqJVeqsh/356-misc-content-tasks).
